### ISSUE TITLE
use_dropout not getting saved in checkpoint ???

### ISSUE THREAD

**Issue by sankaran45** [flairNLP/flair#1628]
I think the use_dropout parameter does not saved as part of state_dict due to bug in
 def _get_state_dict(self) 

as a result of which if we set a dropout layer in sequence tagger, and resume checkpoint training, the behavior changes.

**Comment by alanakbik** [comment]
@sankaran45 thanks for spotting this! I'll fix in a PR!
